src/arcade/potts/agent/cell/PottsCell.java

```diff
@@ -295,6 +295,15 @@ public double getCriticalVolume() {
         return criticalVolume;
     }
 
+    /**
+     * Sets the critical volume of the cell.
+     *
+     * @param newCriticalVolume the new critical volume
+     */
+    public void setCriticalVolume(double newCriticalVolume) {
```

Comment:
Only thing to think about is whether or not we want a subset of classes to have a settable critical volume vs. not.